tests/TestHelpers/OcisConfigHelper.php

```diff
@@ -54,10 +54,18 @@ private static function sendRequest(
 		);
 
 		try {
-			return $client->send($request);
+			$response = $client->send($request);
 		} catch (ConnectException $e) {
 			throw new \Error("Cannot connect to the ociswrapper at the moment, make sure that ociswrapper is running before proceeding with the test run.\n" . $e->getMessage());
+		} catch (GuzzleException $ex) {
+			$response = $ex->getResponse();
```

Comment:
WHat will it throw if it's GuzzleException and response is not null?

Reply:
it will not throw but return the response